var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -24,6 +24,7 @@ class Dd4hep(CMakePackage):
     tags = ['hep']
 
     version('master', branch='master')
+    version('01-17-00', sha256='e56071ce5497517fe56af813828b1a5fab90f0b602b86961a277ca22be215232')
```

Comment:
Is this a typo for:
```suggestion
    version('1.17.0', sha256='e56071ce5497517fe56af813828b1a5fab90f0b602b86961a277ca22be215232')
```
or the version structure changed?

Reply:
I simply copied the output of `spack checksum` without checking this in more detail to be honest. In principle the versioning scheme uses dashes, but that is handled by get `url_from_version`. I will change this to use dots. 

@andresailer, did the dd4hep versioning change, since the patch release is included in this new minor release, which was previously not the case?

Reply:
I was just not careful about how previous tags were named. But the hyphens were always used.

Reply:
I created https://github.com/AIDASoft/DD4hep/releases/tag/v01-17 from the same commit